core/vm/runtime/env.go

```diff
@@ -17,11 +17,16 @@
 package runtime
 
 import (
+	"fmt"
+
 	"github.com/ethereum/go-ethereum/core"
 	"github.com/ethereum/go-ethereum/core/vm"
 )
 
 func NewEnv(cfg *Config) *vm.EVM {
+	if cfg.ChainConfig.IsArbitrum() {
+		panic(fmt.Errorf("chain config has arbitrum enabled"))
```

Comment:
not important and not mandatory - I'd skip the fmt.error, to reduce merge conflicts in the import section.

Reply:
done.